### PR TITLE
Misc - Add Full Body Stiching

### PR DESCRIPTION
**Pull request by Belhun**
**When merged this pull request will:**
- Reintroduce Full Body Stitching to the medical menu, using scripts adapted from an older version of ACE. This adaptation occurs after the HashMap introduction and before the shift to per-body-part stitching.
- Ensure compatibility of the old stitching scripts within KAT medical systems.
- Add actions to ACE_Medical_Treatment_Actions to support Full Body Stitching. Currently, two actions are defined; the first is temporarily commented out to debug issues related to operation time discrepancies.
Things To Still Do:
- [x] Remove Comment out debugging Code
- [x] Fixing stitching is not on a Timer
- [x] Add Setting Toggleing Fullbody Stitching

### IMPORTANT

- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/misc/functions/fnc_fullBodyCanStitch.sqf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * Example:
+ * [player, cursorTarget] call kat_misc_fnc_fullBodyCanStitch
+ *
+ * Public: No
+ */
+
+params ["_medic", "_patient"];
+
+
+if ((GVAR(consumeSurgicalKit) == 2) call FUNC(hasItem)) exitWith { false};
```
Comment (MissHeda) [inline addons/misc/functions/fnc_fullBodyCanStitch.sqf]:
```suggestion
if ((GVAR(consumeSurgicalKit) == 2) call FUNC(hasItem)) exitWith {false};
```

**Review by MissHeda — CHANGES_REQUESTED** [review]

File: addons/misc/functions/fnc_fullBodyCanStitch.sqf (hunk 1 above)
Comment (Belhun) [reply addons/misc/functions/fnc_fullBodyCanStitch.sqf]: ###  Suggestion Applied :)

**Comment by Belhun** [comment]
ToDo:
- [x] Change Nameing Structure of Functions and English Translations From "Full Body Stitching" to "Stitching Full Body"

**Comment by Belhun** [comment]
Allright! Am considering this Ready to go, nothing Else i can think of to Add. welll actually am gonna fix the Order in Prep, cause now its bothering me since i changed the names of the funcions

File: addons/misc/ACE_Medical_Treatment_Actions.hpp [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class ACE_Medical_Treatment_Actions {
+    class FieldDressing;
```
Comment (MissHeda) [inline addons/misc/ACE_Medical_Treatment_Actions.hpp]: Why add FieldDressing here?

File: addons/misc/stringtable.xml [hunk 3]
```diff
@@ -1353,5 +1353,15 @@
             <Spanish>Solto al paciente %1</Spanish>
             <Finnish>Lakkasi kantamisen %1</Finnish>
         </Key>
+        <Key ID="STR_KAT_Misc_Use_SurgicalKitFullBody">
+            <English>Use Surgical Kit(Full Body)</English>
```
Comment (MissHeda) [inline addons/misc/stringtable.xml]:
```suggestion
            <English>Use Surgical Kit (Full Body)</English>
```

File: addons/misc/stringtable.xml [hunk 4]
```diff
@@ -1353,5 +1353,15 @@
...
+            <Japanese>全身を縫合する</Japanese>
+        </Key>
+        <Key ID="STR_KAT_Misc_SETTING_EnableStitchFullBody">
+            <English>Enable Full Body Stitch</English>
```
Comment (MissHeda) [inline addons/misc/stringtable.xml]:
```suggestion
            <English>Enable Full Body Stitching</English>
```

**Review by MissHeda — CHANGES_REQUESTED** [review]

File: addons/misc/ACE_Medical_Treatment_Actions.hpp (hunk 2 above)
Comment (Belhun) [reply addons/misc/ACE_Medical_Treatment_Actions.hpp]: Left over from testing, totally should be removed good find

**Review by MissHeda — APPROVED** [review]

**Comment by Belhun** [comment]
Is there anything anything else I need to do to improve this?

**Comment by MissHeda** [comment]
Did you test it ingame yet?

**Comment by Belhun** [comment]
> Did you test it ingame yet?

Yes, i did. The feature works as intended. The setting is also working.

**Review by MiszczuZPolski — APPROVED** [review]

**Pull request merged**